### ISSUE TITLE
Error occurred while enabling Ships v5.0.1.7

### ISSUE THREAD

**Issue by MuerteInHell** [mosemister/Ships-5-Bukkit-Version#4]
Using MC 1.9 and latest spigot, with the 1.9 version of ships:
[https://www.spigotmc.org/resources/ships-mc-1-8-1-9.2072/update?update=74732](https://www.spigotmc.org/resources/ships-mc-1-8-1-9.2072/update?update=74732)

```
[12:37:46] [Server thread/ERROR]: Error occurred while enabling Ships v5.0.1.7 (Is it up to date?)
java.lang.IllegalArgumentException: Cannot set to an empty path
    at org.apache.commons.lang.Validate.notEmpty(Validate.java:321) ~[minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at org.bukkit.configuration.MemorySection.set(MemorySection.java:163) ~[minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at org.bukkit.configuration.MemorySection.set(MemorySection.java:193) ~[minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at MoseShipsBukkit.Utils.ConfigLinks.MaterialsList.save(MaterialsList.java:240) ~[?:?]
    at MoseShipsBukkit.Ships.onEnable(Ships.java:57) ~[?:?]
    at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:292) ~[minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at org.bukkit.craftbukkit.v1_9_R1.CraftServer.loadPlugin(CraftServer.java:361) [minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at org.bukkit.craftbukkit.v1_9_R1.CraftServer.enablePlugins(CraftServer.java:321) [minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at net.minecraft.server.v1_9_R1.MinecraftServer.t(MinecraftServer.java:411) [minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at net.minecraft.server.v1_9_R1.MinecraftServer.l(MinecraftServer.java:376) [minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at net.minecraft.server.v1_9_R1.MinecraftServer.a(MinecraftServer.java:331) [minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at net.minecraft.server.v1_9_R1.DedicatedServer.init(DedicatedServer.java:269) [minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at net.minecraft.server.v1_9_R1.MinecraftServer.run(MinecraftServer.java:527) [minecraft_server.jar:git-Spigot-f04e043-52c7fc0]
    at java.lang.Thread.run(Unknown Source) [?:1.8.0_66]
```

**Comment by mosemister** [comment]
Thank you for reporting the issue. For some reason i put a "," instead of a "+". this will be fixed in the next update. The issue is caused because Ships is detecting a unknown block and it is trying to add it into the materials list, however that part of the code is for the old Materials list (before Ships 5.0.1.0). 

If you are using Spigot 1.9 and using Ships 5.0.1.7 (MC 1.9) then it should not be detecting custom blocks. 

This issue will close when the next update is open to the public.

**Comment by mosemister** [comment]
This bug is fixed in 5.0.1.8
